<launch>
  <arg name="bag" default=""/>
  <arg name="seq" default="4_12"/>

  <rosparam command="load" file="$(find direct_stereo_slam)/cams/kitti/$(arg seq)/T_stereo.yaml" ns="direct_stereo_slam"/>
  <node name="direct_stereo_slam" pkg="direct_stereo_slam" type="slam_node" output="screen" required="true">
    <param name="topic0" value="/left/image_raw" />
    <param name="topic1" value="/right/image_raw" />
    <param name="calib0" value="$(find direct_stereo_slam)/cams/kitti/$(arg seq)/camera0.txt" />
    <param name="calib1" value="$(find direct_stereo_slam)/cams/kitti/$(arg seq)/camera1.txt" />
    <param name="bag" value="$(arg bag)" />
    <!-- <param name="scale_opt_thres" value="-1" /> -->
  </node>

  </launch>
